oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/LuceneIndexMBeanImpl.java

```diff
@@ -365,6 +369,16 @@ public boolean checkConsistencyOfAllIndexes(boolean fullCheck) throws IOExceptio
         return clean;
     }
 
+    @Override
+    public String performPropertyIndexCleanup(String paths, int batchSize, int sleepPerBatch, int maxRemoveCount) throws CommitFailedException {
+        String result = "PropertyIndexCleaner not enabled";
+        if (propertyIndexCleaner != null) {
+            result = "Removed nodes: " + propertyIndexCleaner.performCleanup(paths, batchSize, sleepPerBatch, maxRemoveCount);
```

Comment:
wouldn't it be more useful to know which paths were actually removed rather than the number of nodes removed?

Reply:
It will remove all nodes that are descendent nodes of the path(s) in the parameters (those that were specified). That could be 1 million nodes in an extreme case. One example is `/oak:index/abc/:index`, and it will remove all nodes below and including that node. I think the number is interesting, however the node name itself is specified in the parameter already.

Reply:
I understood that, it just seemed that "Removed nodes: 14" seemed a bit ambiguous and that adding the paths of what is removed it would provide more info.